qiskit/_quantumprogram.py

```diff
@@ -1157,88 +1140,79 @@ def run_batch(self, qobj_list, wait=5, timeout=120):
-        immediately.
-
-        All input for run comes from qobj.
-
-        Args:
-            qobj_list (list(dict)): The list of quantum objects to run.
-            wait (int): Time interval to wait between requests for results
-            timeout (int): Total time to wait until the execution stops
-            callback (fn(results)): A function with signature:
-                    fn(results):
-                    The results param will be a list of Result objects, one
-                    Result per qobj in the input list.
-        """
-        self._run_internal(qobj_list,
-                           wait=wait,
-                           timeout=timeout,
-                           callback=callback)
+    # def run_async(self, qobj, wait=5, timeout=60, callback=None):
+    #     """Run a program (a pre-compiled quantum program) asynchronously. This
+    #     is a non-blocking function, so it will return immediately.
+
+    #     All input for run comes from qobj.
+
+    #     Args:
+    #         qobj(dict): the dictionary of the quantum object to
+    #             run or list of qobj.
+    #         wait (int): Time interval to wait between requests for results
+    #         timeout (int): Total time to wait until the execution stops
+    #         callback (fn(result)): A function with signature:
+    #                 fn(result):
+    #                 The result param will be a Result object.
+    #     """
+
+    #     def job_done_callback(results):
+    #         """Callback called when the job is done. It basically
+    #         transforms the results to what the user expects and pass it
+    #         to the main thread.
+    #         """
+    #         callback(results[0])  # The user is expecting a single Result
+
+    #     self._run_internal([qobj],
+    #                        wait=wait,
+    #                        timeout=timeout,
+    #                        callback=job_done_callback)
+
+    # def run_batch_async(self, qobj_list, wait=5, timeout=120, callback=None):
+    #     """Run various programs (a list of pre-compiled quantum program)
+    #     asynchronously. This is a non-blocking function, so it will return
+    #     immediately.
+
+    #     All input for run comes from qobj.
+
+    #     Args:
+    #         qobj_list (list(dict)): The list of quantum objects to run.
+    #         wait (int): Time interval to wait between requests for results
+    #         timeout (int): Total time to wait until the execution stops
+    #         callback (fn(results)): A function with signature:
+    #                 fn(results):
+    #                 The results param will be a list of Result objects, one
+    #                 Result per qobj in the input list.
+    #     """
+    #     self._run_internal(qobj_list,
+    #                        wait=wait,
+    #                        timeout=timeout,
+    #                        callback=callback)
```

Comment:
just remove the code, thanks to git history there's no need to comment code anymore :)